keystorage/aws/src/main/java/tech/pegasys/signers/aws/AwsSecretsManager.java

```diff
@@ -68,6 +76,39 @@ static AwsSecretsManager createAwsSecretsManager() {
     }
   }
 
+  public <R> Collection<R> mapSecretsList(
+      final String prefix, final BiFunction<String, String, R> mapper) {
+    final Set<R> result = ConcurrentHashMap.newKeySet();
+    secretsManagerClient.listSecretsPaginator().stream()
```

Comment:
What is default paging size? If the default is less than the max 100 we should configure a custom ListSecretsRequest to minimised the number additional requests needed